data/net/minecraft/world/entity/ai/control/MoveControl.mapping

```diff
@@ -1,4 +1,12 @@
 CLASS net/minecraft/world/entity/ai/control/MoveControl
+	METHOD <init> (Lnet/minecraft/world/entity/Mob;)V
+		ARG 1 mob
+	METHOD hasWanted ()Z
+		COMMENT @return If the mob is currently trying to go somewhere
+	METHOD isWalkable (FF)Z
+		COMMENT @return true if the mob can walk successfully to a given X and Z
```

Comment:
```suggestion
		COMMENT @return {@code true} if the mob can walk successfully to a given X and Z
```

Reply:
Yeah this is a needless semantic difference that does not actually add any readability or usability to the mappings. Please don't add reviews just to be a pedant as it does actually take me time to go through these once a week

Reply:
It's a fair comment, some (if not most) of these reviews are pedantic, that's part of the reason I am writing out the line changes on them. I like the idea of keeping this stuff consistent but it's not really my choice anyway. 

Keep in mind that these in particular are meant as suggestions, you of course have every right not to take this into account given most of it is not stated in the standards.

Reply:
As it stands either way is fine, while a good albeit pendantic suggestion it is not a required change in my book.